RELEASE.md

```diff
@@ -21,28 +21,28 @@ Releases can be triggered in two ways:
    > Integration tests will continue to run in Travis as they currently do, after the automated pull request is created it would run the tests on `prepare-release` branch.
 
 1. The merge will trigger the automatic generation of a new tag using the semantic version provided during the merging of the pull request above.
-
 2. After the the generated tag is pushed to the branch this will trigger the docker build and publish the statics and push the images to Docker Hub. The development and production images would be accessible in docker [https://hub.docker.com/u/oneamex](https://hub.docker.com/u/oneamex)
+3. The statics assets will be published and added to a tag that has been released via the [release static assets action](.github/workflows/docker_release.yml)
+4. We are currently using [https://github.com/release-drafter/release-drafter](https://github.com/release-drafter/release-drafter) to generate release notes. This is a github action that generates draft releases note that can be added to the released tag. Please add the labels specified within [release-drafter](.github/release-drafter.yml) to categorize different pull requests to ensure that any changes are categorized correctly. You can also use the changelog generated for this content.
```

Comment:
does release-drafter have to use labels or can it use commits ? can we auto label based of the commit.
If we take the changes add as part of the changelog do we even need release drafter ?

Reply:
We have some work to address this in a separate PR